python/cuml/test/test_umap.py

```diff
@@ -402,25 +397,19 @@ def get_embedding(n_components, random_state):
         reducer.fit(fit_data, convert_dtype=True)
         return reducer.transform(transform_data, convert_dtype=True)
 
-    if isinstance(random_state, np.random.RandomState):
-        state = random_state.get_state()
-
-    cuml_embedding1 = get_embedding(n_components, random_state)
-
-    if isinstance(random_state, np.random.RandomState):
-        random_state.set_state(state)
-
-    cuml_embedding2 = get_embedding(n_components, random_state)
+    state = copy.deepcopy(random_state)
```

Comment:
Just curious, why the deep copy? Does `random_state` get modified somewhere?

Reply:
Removed.